JS/script.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable class-methods-use-this */
+let bookTitle;
+let bookAuthor;
+const library = document.querySelector('.library');
+const bookList = JSON.parse(localStorage.getItem('bookList')) || [];
+
+function getInformation() {
+  bookTitle = document.getElementById('bookTitle').value;
+  bookAuthor = document.getElementById('bookAuthor').value;
+}
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class BookShelf {
+  constructor() {
+    this.head = null;
+    this.tail = null;
+    this.length = 0;
+  }
+
+  addBook(title, author) {
+    const newBook = new Book(title, author);
+    bookList.push(newBook);
+    localStorage.setItem('bookList', JSON.stringify(bookList));
+    this.get();
+  }
+
+  deleteBook(title) {
+    const resultBooks = bookList.filter((book) => book.title !== title);
+    localStorage.setItem('bookList', JSON.stringify(resultBooks));
+    window.location.reload();
+  }
```

Comment:
Kindly make sure that when you remove a book the correct book is removed even if you have duplicate books. You can add an ID property to make it easier.  :pray: :pray: